test/conformance/helpers/channel_tracing_test_helper.go

```diff
@@ -52,19 +54,22 @@ func ChannelTracingTestHelper(t *testing.T, channelTestRunner common.ChannelTest
 				client := common.Setup(st, true)
 				defer common.TearDown(client)
 
+				// Label namespace so that it creates the default broker.
+				if err := client.LabelNamespace(map[string]string{"knative-eventing-injection": "enabled"}); err != nil {
```

Comment:
It seems you are not actually using the `default` broker in the tests? If that's the case, you can also use the helper function in https://github.com/knative/eventing/blob/d2080374d3ed23abbd25f2878367e94ef1de86b7/test/common/creation.go#L298 to create the required RBAC resources for creating new brokers...
But up to you.

Reply:
Good catch. In fact, I only want the Broker for the Broker test, not the Channel tests. So I removed the annotation from here and use the suggested `CreateRBACResourcesForBrokers` in the Broker setup.